lib/infomon/parser.rb

```diff
@@ -165,6 +175,21 @@ def self.parse(line)
           else
             :noop
           end
+        when Pattern::GoalsDetected
+          @goals_detected = true
+          :ok
+        when Pattern::GoalsEnded
+          if @goals_detected
+            @goals_detected = false
+            respond; respond monsterbold_start
+            respond "You just trained your character.  Lich will gather your updated skills."
+            respond monsterbold_end; respond
```

Comment:
This can be cleaned up a bit using native monsterbold messaging library:
```ruby
respond
respond Lich::Messaging.monsterbold('You just trained your character.  Lich will gather your updated skills.')
respond
```